expertsystem/state/__init__.py

```diff
@@ -1,8 +0,0 @@
-"""All modules related to a quantum mechanical state."""
-
-__all__ = [
-    "combinatorics",
-    "properties",
-]
-
-from . import combinatorics, properties
```

Comment:
I just comment here, since i can't find a better place. Is it possible to abolish this properties module and move the functionality to appropriate modules? Things that have to do with solving can go into solving modules. Things that have to do with ui more into system control or so.

Mainly I want to have the code logically grouped together